DNN Platform/Library/Obsolete/SerializationController.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Entities.Modules.Settings
+{
+    using System;
+    using System.Reflection;
+
+    using DotNetNuke.Abstractions;
+    using DotNetNuke.Common;
+    using Microsoft.Extensions.DependencyInjection;
+
+    [Obsolete("Deprecated in 9.8.0. Use Dependency Injection to resolve 'DotNetNuke.Abstractions.ISerializationManager' instead. Scheduled for removal in v11.0.0.")]
+    public partial class SerializationController
+    {
+        internal static ISerializationManager SerializationManager =>
+            Globals.DependencyProvider.GetRequiredService<ISerializationManager>();
```

Comment:
This should be private otherwise any object in the `DotNetNuke.Library` has access to this reference